lib/ReadingStatsModelQuery.js

```diff
@@ -0,0 +1,63 @@
+import db from '../models';
+
+const { readerStats, profiles, articles } = db;
+/**
+ * @exports ReadingStatsModelQuery
+ * @class ReadingStatsModelQuery
+ * @description Handles all Reading Statistics in the application
+ * */
+class ReadingStatsModelQuery {
+/**
+   *
+   * method to store reading statistics views
+   * @static
+   * @param {integer} readInfo
+   * @returns {json} return json object to user
+   * @memberof ReadingStats
+   */
+  static async createReaderStats(readInfo) {
+    const { articleId, userId } = readInfo;
```

Comment:
No try catch here?

Reply:
I called the method in the ReadingStatController which has a try and catch. I thought even though there is an error the ReadingStatController catch will 'catch' the error. But I will like to get more understanding of it. Thanks